### DISCUSSION TITLE
如何引入 pug-plain-loader 以便支持编译 vue 的 pug 模板 <template lang="pug'> 呢？

### DISCUSSION THREAD

**Discussion by Riant** [NervJS/taro#9792] [Q & A]
如题，翻遍文档各个角落，只有 sass, csso, postcss 之类，却没有 pug 相关的说明，看到一个 issue，貌似已经支持了，所以被直接关闭了（为何不在关闭的时候说一下怎么配置呢？）。

百度了一下，貌似 taro3 确实是已经支持 pug 的了，但评论回复也只是说 “引入pug-plain-loader” 即可，那么怎么引入 “pug-plain-loader” 以支持 pug 编译呢？

谢谢。

另外吐槽一下，scss 之类是直接安装编译依赖之后即可支持，无需配置 loader, 为何 pug 不是同理呢？

**Comment by Chen-jj** [comment]
可以使用 webpack-chain 修改 webpack 配置，加入此 loader 试试
